pages/library/_viewId.vue

```diff
@@ -115,20 +125,13 @@ export default Vue.extend({
       this.loading = false;
     }
   },
-  async beforeMount() {
+  beforeMount() {
     this.setAppBarOpacity({ opaqueAppBar: true });
     this.$nextTick(() => {
       this.$nuxt.$loading.start();
     });
     try {
```

Comment:
Can this try block be removed?

Also what happens if the request in asyncData fails?

Reply:
> Also what happens if the request in asyncData fails?

It essentially does what we've been doing manually. It'll throw an error and show the error page.